doc/ros_api.md

```diff
@@ -111,7 +111,7 @@ Note: errors and alarms which require physical operator intervention (e-stops, e
 
 Type: [motoros2_interfaces/srv/StartTrajMode](https://github.com/yaskawa-global/motoros2_interfaces/blob/d6805d32714df4430f7db3d8ddc736c340ddeba8/srv/StartTrajMode.srv)
 
-Attempt to enable servo drives and activate trajectory mode, allowing the action server (`follow_joint_trajectory`, see below) to execute incoming `FollowJointTrajectory` action goals.
+Attempts to set cycle mode, enable servo drives, and activate trajectory mode, allowing the action server (`follow_joint_trajectory`, see below) to execute incoming `FollowJointTrajectory` action goals.
```

Comment:
we're probably not going to do anything about it, but it's a bit 'mysterious' to have this state it's going to *set cycle mode* without any motivation. Enabling the servos and activating trajectory mode all makes sense, but changing cycle mode seems odd.

Reply:
Are you saying that you'd prefer we remove this text? 

I think it's implied that if this service is performing a particular action, then there's a good reason for it.

Reply:
> Are you saying that you'd prefer we remove this text?

No, just that the part about this 'attempting to set cycle mode' is a bit cryptic. The other actions it performs 'make sense': can't move the robot without the servos being enabled or trajectory mode being activated.

But then it also *sets cycle mode*, which to me at least, is far from clear why.

But perhaps this is too much of a nit.

> I think it's implied that if this service is performing a particular action, then there's a good reason for it.

Of course. I just always like to understand *why* something happens.